src/Client/Client.csproj

```diff
@@ -2,10 +2,12 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
+
   <ItemGroup>
-    <PackageReference Include="System.IO.Pipelines" Version="4.5.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
why are you removing System.IO.Pipelines? the point of this exercise is to keep using this API but not echo

Reply:
It was a misunderstanding and I will soon update the PR